spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -220,6 +221,51 @@
       to include('Verified within value must be at least 30 days or older')
   end
 
+  it 'sends the user through idv again via verified_within param' do
+    client_id = 'urn:gov:gsa:openidconnect:sp:server'
+    user = user_with_2fa
+    _profile = create(:profile, :active,
+                      verified_at: 60.days.ago,
+                      pii: { first_name: 'John', ssn: '111223333', dob: '1970-01-01' },
+                      user: user)
+
+    token_response = sign_in_get_token_response(
+      user: user,
+      acr_values: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
+      client_id: client_id,
+      redirect_uri: 'http://localhost:7654/auth/result',
+      scope: 'openid email profile',
+      verified_within: '30d',
+      proofing_steps: proc do
+        complete_all_doc_auth_steps
+
+        fill_out_phone_form_mfa_phone(user)
+        click_idv_continue
+
+        fill_in :user_password, with: Features::SessionHelper::VALID_PASSWORD
+        click_continue
+
+        acknowledge_and_confirm_personal_key(js: false)
```

Comment:
I'm not super excited that I had to write out the proofing steps like this...it makes for a long acceptance test, which is a more fragile acceptance test.... any suggestions for how to better check that we actually *do* send you through the proofing flow...other than to go through it?

Reply:
Check out `complete_idv_steps_with_usps_before_confirmation_step` in `idv_step_helper`. That should get you all but the last mile. We could probably add another method, like `complete_all_idv_steps` there.

Reply:
Ok so I gave that a shot, and unfortunately that one jumps around going to different URLs directly, so somehow the user is being prompted to log in again with user/pass....and would require some bigger surgery to either skip the jumping or to add a log back in step.

Is it ok to leave this as-is for the time being since it works and is readable?